pkg/controller/upload-controller.go

```diff
@@ -21,18 +21,29 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/pkg/errors"
+
```

Comment:
I kind of hate to bring this up at this point but I wonder if it makes more sense to have a dedicated cdiconfig controller rather than rolling this into the upload controller?  I could imagine exposing more stuff through cdiconfig.  Could also allow users to set configuration params as well.  What do others think?

Reply:
I tend to agree with you. We took the upload controller approach since we only have upload proxy related stuff but I think it needs its own controller.